### PR TITLE
Full SSR pages (applies to authentication app)

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: react/pages/authentication/ResetPasswordPage/index.js [hunk 1]
```diff
@@ -18,13 +20,23 @@ export default class ResetPasswordPage extends Component {
...
+                <LoadingIndicator />
+              </CenteringBox>
+            );
+          }
 
           if (error) {
             return (
               <CenteringBox p={7} flexDirection="column">
+                <Head>
+                  <title>Sorry</title>
```
Comment (broskoski) [inline react/pages/authentication/ResetPasswordPage/index.js]: Are.na / Sorry

Comment (dzucconi) [reply react/pages/authentication/ResetPasswordPage/index.js]: That's what it renders

File: react/components/UI/Head/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet-async';
+
+const Head = ({ children, ...rest }) => (
+  <Helmet defaultTitle="Are.na" titleTemplate="Are.na / %s" {...rest}>
```
Comment (dzucconi) [inline react/components/UI/Head/index.js]: @broskoski Template here

File: react/pages/authentication/ResetPasswordPage/index.js (hunk 1 above)
Comment (broskoski) [reply react/pages/authentication/ResetPasswordPage/index.js]: I know, it's funny

Comment (dzucconi) [reply react/pages/authentication/ResetPasswordPage/index.js]: Are.na is sorry

File: react/components/UI/Page/index.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const GOOGLE_ANALYTICS_TRACKING_TAG = `
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+`;
```
Comment (dzucconi) [inline react/components/UI/Page/index.js]: Ah this probably needs to set the tracking ID?

Comment (dzucconi) [reply react/components/UI/Page/index.js]: Hm, this is annoying. Let's chat real quick.

**Comment by dzucconi** [comment]
So lil thing I noticed that we'll have to figure out how to deal with: we should probably stop using/come up with a different method of handling sharify data that's set within routes. We're kind of using it in a  to just shovel data around globally, which is breaking on client-side route changes. You can see this in production by loading /sign_up server-side, then navigating to /log_in. You'll be redirected to /welcome on login.

**Pull request merged**